### PR TITLE
fix(blade): add overlay color token

### PR DESCRIPTION
**Pull request by kamleshchandnani**

### CODE REVIEW

**Review by saurav12 — APPROVED** [review]
This looks good to me! 👍
Updating token on Figma as well.

**Review by saurav12 — DISMISSED** [review]

File: packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ const paymentLightTheme: Theme = {
         },
       },
     },
+    overlay: { background: globalColors.neutral.blueGrayLight.a200 },
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme.ts:83]: Why is prettier not moving this to a new line? 😰
```suggestion
    overlay: { 
    	background: globalColors.neutral.blueGrayLight.a200 
    	},
```

Comment (kamleshchandnani) [reply packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme.ts:83]: we need to manually move it for some cases 🤕

Comment (chaitanyadeorukhkar) [reply packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme.ts]: So much better now ❤️

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Pull request merged**